.claude/agents/pr-reviewer.md

```
@@ -1,139 +1,395 @@
+    next()
+  } else {
+    return res.status(403).json({ error: 'Insufficient privileges' })
+  }
+}
+```
+````
 
-- **Accepted**: [List of valid CodeRabbit findings retained]
-- **Filtered**: [Count of dismissed suggestions and why]
-- **Added**: [New issues CodeRabbit missed]
+#### 2. [OWASP-A03] SQL Injection Vector
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)